src/asm/mod.rs

```diff
@@ -0,0 +1,69 @@
+#[link(name = "x86_64_asm", kind = "static")]
+extern "C" {
+    #[cfg_attr(target_env = "gnu", link_name = "_x86_64_asm_interrupt_enable")]
```

Comment:
What is this about? Why is the leading `_` not used on non-gnu platforms?

Reply:
On MacOs, the extern link_name gets an automatic "_" . On Linux/GNU it does not.

Reply:
Thanks for clarifying!